bundles/fluent-en_GB/atrium-fluent-en_GB/build.gradle.kts

```diff
@@ -1,6 +1,11 @@
 description = "Represents a convenience module which merely bundles dependencies."
 
+val kotestVersion: String by rootProject.extra
+
 kotlin {
+    tasks.withType<Test> {
+        useJUnitPlatform()
+    }
```

Comment:
you can remove this one again, this is covered by the includeEngines in build.gradle